topologies/atedut_3.testbed

```diff
@@ -0,0 +1,46 @@
+# proto-file: github.com/openconfig/ondatra/blob/main/proto/testbed.proto
+# proto-message: ondatra.Testbed
+
+# This testbed provides 2 links between a DUT and an ATE connected
```

Comment:
Nit: this should say "3 links"

Reply:
@dan-lepage , I have removed testbed file which is not actually not needed foe this test. My changes are only with .go files. Hence removed .testbed file in this PR. Thanks for pointing it.